metron-deployment/packaging/ambari/metron-mpack/src/main/resources/common-services/METRON/CURRENT/configuration/metron-enrichment-env.xml

```diff
@@ -129,70 +145,156 @@
   <property>
     <name>enrichment_topology_max_spout_pending</name>
     <description>Spout Max Pending Tuples for the Enrichment Topology</description>
-    <value/>
-    <display-name>Enrichment Max Pending</display-name>
+    <value>500</value>
```

Comment:
This seems super low..shouldn't we leave this unspecified and default to the storm default?

Reply:
This is about tuning the user's day 1 experience.  Specifically what I have seen is on day 1 users push some telemetry into a topic, start up the topology, and the topology just blows-up.  They have no idea why because they are on day 1.   That is a bad experience.

The topology blows-up because on day 1 the user is running with a topology that has NOT been tuned.  The topology blows-up because when this value is undefined there is no back pressure.  We need some back pressure here to avoid the day 1 blow-up.

I'd prefer a day 1 user be able to start up the topology and see it actually work, albeit slowly, under most conditions.  A user running against heavier load is more likely to be experienced and should have more knowledge to know that this value needs increased.

Reply:
But this is definitely not necessary for the task at hand; METRON-1499.  

If this is contentious, I can certainly pull this change out of this PR.  I could then submit a separate PR where we can debate this point.  I don't want to hang up this PR for something unrelated.  

I'm good with either way.

Reply:
+1 for a sane default

Reply:
I can get behind a sane default.  I retract my comment.

Reply:
Ok, that works.  It certainly deserved an explanation from me, so glad you called that out.